src/templates/next/components/OrderedList/OrderedList.tsx

```diff
@@ -0,0 +1,25 @@
+import type { OrderedListProps } from "~/common"
+import UnorderedList from "../UnorderedList"
+import Paragraph from "../Paragraph"
+
+const OrderedList = ({ start, items }: OrderedListProps) => {
+  return (
+    <ol className="list-decimal ps-8 my-1" start={start}>
+      {items.map((item) => {
+        if (typeof item === "string") {
+          return (
+            <li key={Math.random()} className="[&_p]:inline pl-2 my-1">
```

Comment:
should we add a TODO here for when we think about rendering the components in the CMS? question applies for unordered list too
For the static site itself I guess it doesnt matter since the key will never change after build

Reply:
Hmm I'm a bit more inclined to leave it out first, cos I feel the decisions made here should not be influenced by how the CMS would handle it. I think we can look at everything again when we start building the CMS side of things.